nh_ews/tests/common/nh_clinical_test_utils.py

```diff
@@ -10,6 +10,50 @@ class NhClinicalTestUtils(AbstractModel):
     _name = 'nh.clinical.test_utils'
     _inherit = 'nh.clinical.test_utils'
 
+    def get_open_obs(self, patient_id=None, data_model=None, user_id=None):
+        api_model = self.pool['nh.eobs.api']
```

Comment:
Docstring here would be good, particularly to explain why the `assign` call is necessary.